portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardAuditGraph.testcase

```diff
@@ -588,72 +588,80 @@ definition {
 	@description = "LPS-133797 When the user selects the option Available, the modal lists all the existing Global Vocabularies"
 	@priority = "5"
 	test CheckAvailableVocabulariesDisplayed {
-		WebContentNavigator.openWebContentAdmin(siteURLKey = "global");
+		task ("Create 2 new vocabularies") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "global");
 
-		ProductMenu.gotoPortlet(
-			category = "Categorization",
-			panel = "Site Administration",
-			portlet = "Categories");
+			ProductMenu.gotoPortlet(
+				category = "Categorization",
+				panel = "Site Administration",
+				portlet = "Categories");
 
-		Vocabulary.addCP(vocabularyName = "Vocabulary Name 1");
+			Vocabulary.addCP(vocabularyName = "Vocabulary Name 1");
 
-		Vocabulary.addCP(vocabularyName = "Vocabulary Name 2");
+			Vocabulary.addCP(vocabularyName = "Vocabulary Name 2");
+		}
 
```

Comment:
Can you use a JSON macro to create these vocabularies?

Reply:
Done!